src/App.tsx

```diff
@@ -1,61 +1,81 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+import { SortType, sortData } from './functions';
 
 export const App: React.FC = () => {
+  const [sortField, setSortField]
+    = useState<SortType>(SortType.DEFAULT);
```

Comment:
![image](https://github.com/mate-academy/react_list-of-goods-hooks/assets/96409563/60af1adb-dcee-4909-8a1b-d50147dcea29)